ui/app/modifiers/code-mirror.js

```diff
@@ -49,6 +49,8 @@ export default class CodeMirrorModifier extends Modifier {
         screenReaderLabel: this.args.named.screenReaderLabel || '',
       });
 
+      editor.focus();
```

Comment:
We should do this with the {{autofocus}} modifier; otherwise in situations with multiple editors on a page elsewhere, the one at the bottom of the page will take focus.